R/bin_link.R

```diff
@@ -6,6 +6,8 @@ readRAST <- function(vname, cat=NULL, ignore.stderr=get.ignore.stderrOption(),
 	NODATA=NULL, plugin=get.pluginOption(), mapset=NULL, 
         useGDAL=get.useGDALOption(), close_OK=TRUE, drivername="GTiff",
         driverFileExt=NULL, return_SGDF=TRUE) {
+    .Deprecated(new="read_RAST", package="rgrass7", old="readRAST",
+        msg="Package rgrass7 transitioning to package rgrass for GRASS 8")
```

Comment:
IIUC this message will mask the default message saying which function is replaced by which, no? Confirmed, see below. Is this intentional?

```
>   writeRAST(nc_basic, "sqdemSP", zcol="sqdem", flags=c("quiet", "overwrite"))
Warning message:
Package rgrass7 transitioning to package rgrass for GRASS 8 
```

A more explicit alternative - likewise for the others:

```suggestion
    .Deprecated(new="read_RAST", package="rgrass7", old="readRAST",
        msg="Package rgrass7 is transitioning to package rgrass for GRASS 8.\n 'readRAST' is deprecated. Use 'read_RAST' instead.")
```